calendars/skimage.yaml

```diff
@@ -1,16 +1,27 @@
 name: scikit-image Community Calendar
 timezone: America/Los_Angeles
 events:
+  - summary: scikit-image Community Call (Europe/Africa)
+    description: |
+      The scikit-image Community Call. Friendly for Europe/Africa.
+
+      Meeting notes: https://hackmd.io/@alexdesiqueira/HyyHcIjQ9
```

Comment:
FYI, if wanted in HackMD you can actually rename the link. Click on the HyyHc... and you can put something more readable. As you wish OC 😃

Reply:
:thinking: yeah, I remember that existed, Pamphile! I didn't find the option though :laughing:

Reply:
If you're admin on the page you should see this. In the example I have typed in the box `scipy-meetup`
<img width="367" alt="Screenshot 2022-06-07 at 00 09 38" src="https://user-images.githubusercontent.com/23188539/172257559-503edef9-3059-4338-bd34-fe39e04aea03.png">

Reply:
@tupui aha! Of course, it was hidden in the share options :laughing: Thank you!